apps/src/lib/config/genesis.rs

```diff
@@ -200,6 +201,16 @@ pub mod genesis_config {
         /// Tendermint node key is used to derive Tendermint node ID for node
         /// authentication
         pub tendermint_node_key: Option<HexString>,
+        /// Validator email (required)
+        pub email: String,
```

Comment:
@s0uk4 @Fraccaman because this is required, this will be a breaking change for testnet setup - you'll need to ask all validators to add this

Reply:
It can just be an empty string. We can't validate the email anyway during the submission.

Reply:
So based on that I think we can also make this an option too. Ie, someone can make their email `potatoe@potatoe@potatoe.com`, which is equivalent to None really.

Reply:
even if this is true, that's the validator's problem, not ours. they need to put an email in for the case where the network needs them to respond to potential security issues. asking for an email field helps us, but it's also a favor to the valds themselves.

if they put in an empty string rather than the proper email and there is a breaking change or risk requiring validators to respond, and they don't see discord /or/ the email, they will go down/put the network at risk and be penalized, which means they will lose money. there is incentive for them not to defect here.

Reply:
It's unclear to me if @s0uk4 is convincing others here, so can we get some clear consensus on how the email should be done? Atm, it is required, and providing an empty string for this arg will return an error, since empty strings are used to indicate that one wants to remove metadata from storage (i.e. if I want to delete my validator website, I pass `--website ""` at the CLI).